build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -3,9 +3,12 @@
 
   <ItemGroup>
     <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
-    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
+    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg"
+                         Exclude="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.symbols.nupkg" >
       <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
     </NupkgsForPublishing>
+    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.symbols.nupkg" />
+    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.SDK.Symbols.*.symbols.nupkg" />
```

Comment:
We chatted some offline about how it seems weird that this package only has `.ni.pdb` files, but it doesn't indicate which platform crossgen they came from. It seems like this symbol package needs to be runtime-specific.

Reply:
@nguerrera 
Can you speak to Davis' question.

Reply:
My follow-up questions are:
1. What is the correct name convention of this symbols.nupkg? Currently it is: "Microsoft.DotNet.SDK.Symbols.[SDK version].symbols.nupkg"
2. Do we need to publish a symbols.nupkg from all the RIDs we build?

Reply:
Yes, we need one for each RID we build (ship).

Reply:
Not sure about naming convention. As long as they all get to symbol server, I'm satisfied. This is more a question for @dagood and team.

Reply:
As far as prodcon goes, naming doesn't matter as long as they're unique. You can follow Core-Setup's convention, though:
```
runtime.osx-x64.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
runtime.rhel.6-x64.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
runtime.win-arm.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
runtime.win-arm64.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
runtime.win-x64.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
runtime.win-x86.Microsoft.NETCore.App.2.1.0-preview1-26218-02.symbols.nupkg
...
```

Reply:
Good catch, btw, @dagood.

Reply:
That's thanks to John showing me a testing location to look at--wouldn't have seen it from the PR. :+1: